src/appleseed.bench/mainwindow/mainwindow.cpp

```diff
@@ -106,26 +107,24 @@ namespace bench {
 // MainWindow class implementation.
 //
 
-#ifndef NDEBUG
-#define FAST_DEBUG_MODE
+// Use a simpler scene and lower rendering quality in debug builds.
+#ifdef APPLESEED_DEBUG
+    #define FAST_DEBUG_MODE
```

Comment:
🏆

Reply:
I didn't knew we had this kind of feature :o

Reply:
Without it running appleseeed.bench in debug is not an option...